bin/mv.js

```diff
@@ -0,0 +1,239 @@
+            return copy(sourceWebtask, data, target);
+        })
+        .then(function () {
+            return sourceWebtask.remove();
+        });
+}
+
+function equal(sourceParams, targetParams) {
+    return _.isEqual(sourceParams, {
+        name: targetParams.name,
+        container: targetParams.container || sourceParams.container,
+        profile: targetParams.profile || sourceParams.profile
+    });
+}
+
+function copy(webtask, data, target) {
+    debug('copy: webtask=%j, data=%j, target=%j',
+        _.omit(webtask, 'token'), data, target);
+
+    if (!data.jtn) {
+        throw Cli.error.cancelled('Not a named webtask.');
+    }
+
+    let claims = cloneWebtaskData(data);
+    claims.jtn = target.name || claims.jtn;
+    claims.ten = target.container || claims.ten;
+
+    let pendingCreate;
+    if (target.profile) {
+        pendingCreate = loadProfile(target.profile)
+            .then(function (profile) {
+                target.profile = profile;
+                target.container = profile.container;
+                claims.ten = target.container || claims.ten;
+                return target.profile.createRaw(claims);
+            });
+    } else {
+        target.profile = webtask.sandbox;
+        target.container = webtask.sandbox.container;
+        pendingCreate = target.profile.createRaw(claims);
+    }
+
+    return pendingCreate
+        .then(function () {
+            target.name = claims.jtn;
+            return moveCronJob(webtask.sandbox, data.jtn, target, {
+                verify: webtask.token
+            });
+        })
+        .then(function () {
+            return copyStorage(webtask, target);
+        })
+        .catch(function (err) {
+            throw Cli.error.cancelled('Failed to create webtask. ' + err);
+        });
+}
+
+function loadProfile(name) {
+    debug('loadProfile: name=%s', name);
+
+    let config = new ConfigFile();
+
+    return config.getProfile(name);
+}
+
+function moveCronJob(profile, name, target, options) {
+    debug('moveCronJob: profile=%j, name=%s, target=%j, options=%j',
+        _.omit(profile, 'token'), name, target, options);
+
+    return coroutine(function*() {
+        let job;
+
+        try {
+            job = yield profile.getCronJob({
+                name: name
+            });
+        } catch (err) {
+            if (err.statusCode === 404) {
+                return;
+            }
+            throw err;
+        }
+
+        let webtask = yield profile.getWebtask({
+            name: target.name,
+            container: target.container
+        });
+
+        if (_.get(options, 'verify') && job.token !== options.verify) {
+            console.log(chalk.bold('* Warning: failed to verify the cron job token (no match).'));
+        }
+
+        yield target.profile.createCronJob({
+            name: target.name,
+            container: target.container,
+            token: webtask.token,
+            schedule: job.schedule
+        });
```

Comment:
I think you also need to set the `state` to be the same as in the originating webtask.